raiden-dapp/src/components/GeneralMenu.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <div class="general-screen-menu">
+    <v-row class="general-screen-menu__account-details" no-gutters>
+      <v-col cols="12">
+        <div class="general-screen-menu__account-details--title">
+          {{ $t('general-menu.account-details') }}
+        </div>
+      </v-col>
+      <v-col cols="2">
```

Comment:
I think the idea of `<v-row>` and `<v-col>` is to have a `<v-row>` for each visual row, so doing something like this:
```
<v-row>
<v-col cols="12">...</v-col>
</v-row>
<v-row>
<v-col cols="2">...</v-col>
<v-col cols="10">...</v-col>
</v-row>
<v-row>
<v-col cols="2">...</v-col>
<v-col cols="10">...</v-col>
</v-row>
...
```

might be better and easier to understand

Reply:
> Good job and sorry for the review delay (didn't see it's ready)
> Only added a couple of minor things, feel free to skip them if you think they're unnecessary

Thank you for looking at it. I will address the feedback and make changes accordingly, that's how I learn how things can be done better. :)